### ISSUE TITLE
100% mAP on test set

### ISSUE THREAD

**Issue by xiang-burlington** [TaoRuijie/TalkNet_ASD#33]
Evaluated the performance on test set with command below:
python trainTalkNet.py --dataPathAVA AVADataPath --evaluation --evalDataType test

And the result was:

03-23 11:28:11 Model para number = 15.01
Model pretrain_AVA.model loaded from previous state!
 45%|████████████████████████████████████████████████████████████████████████▊                                                                                        | 9663/21361 [12:25<19:26, 10.03it/s]WARNING:root:frame length (556) is greater than FFT size (512), frame will be truncated. Increase NFFT to avoid.
...
21361/21361 [25:31<00:00, 13.94it/s]
mAP 100.00%

Obviously that's too good to be true. Tried different models but the result was the same. I wonder how to fix it.
BTW: the evaluation result on validation set was right.

**Comment by TaoRuijie** [comment]
No lable in the test set, we can not get the mAP in this set by using that.

Please check the AVA website

**Comment by xiang-burlington** [comment]
I see. Thanks for the explanation.
